server/elm/src/Translate.elm

```diff
@@ -384,6 +395,26 @@ translationSet transId =
             , kirundi = Nothing
             }
 
+        PopulationSelectionOption selectionOption ->
+            case selectionOption of
+                SelectionOptionGlobal ->
+                    { english = "Gloabal"
+                    , kinyarwanda = Nothing
+                    , kirundi = Nothing
+                    }
+
+                SelectionOptionDemographics ->
+                    { english = "Demographics"
+                    , kinyarwanda = Nothing
+                    , kirundi = Nothing
+                    }
+
+                SelectionOptionHealthCenter ->
+                    { english = "Health Center"
+                    , kinyarwanda = Nothing
```

Comment:
```suggestion
, kinyarwanda = Just "Ikigo Nderabuzima"
```

Reply:
My point is that from the client, we are able to populate translations partially.

Reply:
Well, right now, there's no requirement to have the APP translated, so I don't see a reason to spend time on it.
If translation requirement will come, we'll probably outsource the task to someone for TIP team, as we usually do.